dangermattic.gemspec

```diff
@@ -7,7 +7,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'dangermattic/gem_version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'dangermattic'
+  spec.name          = 'danger-dangermattic'
```

Comment:
I'm slightly surprised that rubocop is ok with the name of the `.gemspec` file differs from the value of the `spec.name` here…
We'll have to rename the file to `danger-dangermattic.gemspec` anyway if we want the logic in `gem-push.sh` relying on `$GEM_NAME` to work as expected.